kaggle/api/kaggle_api_extended.py

```diff
@@ -2853,7 +2840,8 @@ def model_get_cli(self, model, folder=None):
       data['slug'] = model_ref_split[1]
       data['title'] = model.title
       data['subtitle'] = model.subtitle
-      data['isPrivate'] = model.isPrivate # TODO Add a test to ensure default is True
+      data[

```

Comment:
no action required, but line breaking due to a comment is kind of ugly.  Maybe something to fix with the autoformatter some day (only if it's easy).

Reply:
Reworded to keep on one line.